src/training/train.py

```diff
@@ -20,28 +22,66 @@ def train_one_epoch(model_video, data, epoch, optimizer, scheduler, args, tb_wri
         world_size=1,
         use_horovod=False,
     )
+
     dataloader = data["train"].dataloader
+    if args.image_data:
```

Comment:
for now I guess this is fine but ideally we want to move this into data.py (I know it's bad) and only create it once in main.py then pass into this function